indigo_libs/indigo_timer.c

```diff
@@ -47,7 +47,7 @@
 //}
 //#endif
 
-#define utc_time(ts) clock_gettime(CLOCK_REALTIME, ts)
```

Comment:
CLOCK_MONOLITHIC is definitely wrong here - utc_time(ts) should return real time not the monolithic clock